### DISCUSSION TITLE
Error launching webui.bat

### DISCUSSION THREAD

**Discussion by leandro9000** [vladmandic/sdnext#634] [Q&A]
When I start the webui.bat, I get this error: NameError: name 'webui' is not defined

![image](https://user-images.githubusercontent.com/37279315/235327555-f8b3de2f-80b9-42f6-973e-5860e4c48252.png)

**Comment by vladmandic** [comment]
sometimes i'm amazed by speed of this community - repo was broken this time for less than 1min and you caught it :)
anyhow, its fixed...

**Reply by leandro9000** [reply]
hahah, sorry! I made a: git pull, and now is working!
